hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -0,0 +1,1462 @@
+    metric-units: milliseconds
+    metric-type:  counter
+  - name: Used Heap Memory
+    object-name:  java.lang:type=Memory
+    attribute:    HeapMemoryUsage#used
+    interval:     30
+    time-units:   seconds
+    metric-units: bytes
+    metric-type:  gauge
+  - name: Aggregate GC Collection Time
+    object-name:  "java.lang:type=GarbageCollector,name=*"
+    attribute:    CollectionTime
+    interval:     30
+    time-units:   seconds
+    metric-units: milliseconds
+- name: Memory Pool Metrics
+  metric-jmx:
+  - name: Initial
+    attribute:    Usage#init
+    interval:     2
+    time-units:   minutes
+    metric-units: bytes
+    metric-type:  gauge
+  - name: Used
+    attribute:    Usage#used
+    interval:     1
+    time-units:   minutes
+    metric-units: bytes
+    metric-type:  gauge
+  - name: Committed
+    attribute:    Usage#committed
+    interval:     1
+    time-units:   minutes
+    metric-units: bytes
+    metric-type:  gauge
+  - name: Max
+    attribute:    Usage#max
+    interval:     2
+    time-units:   minutes
+    metric-units: bytes
+    metric-type:  gauge
+
+avail-set-jmx:
+- name: Runtime Availability
+  avail-jmx:
+  - name: VM Availability
+    interval:   30
+    time-units: seconds
+    attribute:  StartTime
+    up-regex:   "[0123456789]+"
+- name: Memory Pool Availability
+  avail-jmx:
+  - name: Memory Pool Availability
+    interval:   30
+    time-units: seconds
+    attribute:  Valid
+    up-regex:   "[tT].*"
+
+resource-type-set-jmx:
+- name: Main
+  resource-type-jmx:
+  - name: Runtime MBean
+    resource-name-template: JMX [%_ManagedServerName%][%type%]
+    object-name: java.lang:type=Runtime
+    metric-sets:
+    - Runtime Metrics
+    avail-sets:
+    - Runtime Availability
+    resource-config-jmx:
+    - name: OS Name
+      attribute:   Name
+      object-name: java.lang:type=OperatingSystem
+    - name: Java VM Name
+      attribute:   VmName
+- name: Memory Pool
+  resource-type-jmx:
+  - name: Memory Pool MBean
+    parents:
+    - Runtime MBean
+    resource-name-template: JMX [%_ManagedServerName%] %type% %name%
+    object-name: "java.lang:type=MemoryPool,name=*"
+    metric-sets:
+    - Memory Pool Metrics
+    avail-sets:
+    - Memory Pool Availability
+    resource-config-jmx:
+    - name: Type
+      attribute: Type
+- name: Hawkular
+  resource-type-jmx:
+  - name: Hawkular WildFly Agent
+    resource-name-template: "Hawkular WildFly Agent"
+    object-name: "org.hawkular:type=hawkular-javaagent"
+    resource-config-jmx:
+    - name: Immutable
+      attribute: Immutable
+    - name: In Container
+      attribute: InContainer
+    operation-jmx:
+    - name: Start
+      internal-name: start
+    - name: Stop
+      internal-name: stop
+    - name: Status
+      internal-name: status
+    - name: Inventory Discovery Scan
+      internal-name: fullDiscoveryScan
+    - name: Inventory Report
+      internal-name: inventoryReport
+
+# MANAGED SERVERS
+
+managed-servers:
```

Comment:
@josejulio the agent now supports the ability to wait for certain resources before starting to monitor the server (see https://issues.jboss.org/browse/HWKAGENT-165). This helps with the problem of the delay=XX not being long enough and the agent starting to monitor too early. So for these local-dmr, remote-dmr entries, add a "wait-for" to wait for the "/" resource. For the local-jmx and remote-jmx, we can put in a "wait-for" on the mbean "java.lang:type=Runtime". It looks like:

```
local-dmr:
  ...the other config properties here...
  wait-for:
  - name: /
```

or for jmx:

```
local-jmx:
  ...the other config properties here...
  wait-for:
  - name: java.lang:type=Runtime
```

Same thing for the remote-dmr and remote-jmx.